src/lidar/instructions_bgflood.json

```diff
@@ -31,7 +31,7 @@
 				}
 			},
 			"linz": {
-				"key": "65cf810763bc49d0beacb17f18b5ce53",
```

Comment:
In future I would remove this entirely. In my test instructions I do not that the "key" label. I just add it using dotenv instead of replace it using dotenv

Reply:
removed the "key" label from the instruction file